tekton-catalog/pipeline-loops/go.mod

```diff
@@ -3,9 +3,13 @@ module github.com/kubeflow/kfp-tekton/tekton-catalog/pipeline-loops
 go 1.13
 
 require (
+	github.com/cenkalti/backoff v2.2.1+incompatible
+	github.com/go-sql-driver/mysql v1.6.0
 	github.com/google/go-cmp v0.5.6
 	github.com/hashicorp/go-multierror v1.1.1
+	github.com/jinzhu/gorm v1.9.16
```

Comment:
Do we have to use `jinzhu/gorm`? or can we use `gorm.io/gorm` instead?

Reply:
nvm, go-gorm only works for v2 and above. KFP also has this dependency so should be fine.